erpnext/accounts/doctype/accounts_settings/accounts_settings.py

```
@@ -91,7 +91,7 @@ class AccountsSettings(Document):
 		merge_similar_account_heads: DF.Check
 		over_billing_allowance: DF.Currency
 		preview_mode: DF.Check
-		receivable_payable_fetch_method: DF.Literal["Buffered Cursor", "UnBuffered Cursor", "Raw SQL"]
+		receivable_payable_fetch_method: DF.Literal["Buffered Cursor", "UnBuffered Cursor"]
```

Ruff (0.15.12)

[error] Syntax error in forward annotation: Unexpected token at the end of an expression

(F722)